azurerm/resource_arm_shared_image_version_test.go

```diff
@@ -217,6 +217,27 @@ resource "azurerm_shared_image_version" "test" {
 }
 `, template)
 }
+func testAccAzureRMSharedImageVersion_imageVersionZrs(rInt int, location, username, password, hostname string) string {
```

Comment:
could we add a test for this in the Resource too, provisioning this in a standalone test (as in the other cases above)